typescript/looker/rtl/constants.ts

```diff
@@ -45,3 +45,5 @@ export const matchModeString = `(^application\\/.*(\\bjson\\b|\\bxml\\b|\\bsql\\
  * @type {string} Regular expression for matching Content-Type headers
  */
 export const matchModeBinary = '^image\\/|^audio\\/|^video\\/|^font\\/|^application\\/|^multipart\\/'
+
+export const isTrue = (value: string) => /true|1/i.test(value)
```

Comment:
This looks great to me, however it's worth mentioning that in the python version we are matching a few more options. `"yes", "y", "true", "t", "1"` are mapped to true and `"no", "n", "false", "f", "0"` are mapped to false. If nothing is specified, we default to true as well.

Reply:
Thanks for the conformance note. I can update `isTrue` for these other patterns. I'll take a look at how you're using `isFalse` also and set up the defaulting function.